package.json

```
@@ -133,6 +125,9 @@
     "tldts-experimental": "5.7.75",
     "webextension-polyfill": "0.9.0"
   },
+  "overrides": {
+    "ansi-html": "0.0.8"
```

To be more specific, I think this would be equivalent to what we had before the change to `overrides`:
```suggestion
    "ansi-html": "mahdyar/ansi-html-community#v0.0.8"
```